pact/provider/assertj/src/main/java/org/arquillian/algeron/pact/provider/assertj/ProviderUrlAssert.java

```diff
@@ -0,0 +1,19 @@
+package org.arquillian.algeron.pact.provider.assertj;
+
+import java.net.URL;
+
+public class ProviderUrlAssert {
```

Comment:
I see that in near future we'll need more parameters, need to figure out but it is a possibility when we start dealing with https.